### PR TITLE
docs: update recommended plugin for vscode

### PR DESCRIPTION
**Pull request by unbyte**
The original recommendation is out of date - axetroy/vscode-deno is deprecated

### CODE REVIEW

File: docs/getting_started/setup_your_environment.md [hunk 1]
```diff
@@ -43,10 +43,9 @@ http imports, and most editors and language servers do not natively support this
 at the moment, many editors will throw errors about being unable to find files
 or imports having unnecessary file extensions.
 
-The community has developed extensions for some editors to solve these issues:
+We have developed extensions for some editors to solve these issues:
```
Comment (piscisaureus) [inline docs/getting_started/setup_your_environment.md:46]: Why this change?

Comment (unbyte) [reply docs/getting_started/setup_your_environment.md:46]: Use `we` to refer to `community` can make beginners realize that this plugin is the best choice ( because it sounds like more official than directly `community`).
In fact, in VS Code plugin store, there are two plugin having the same name `deno`. They are both excellent plugin for deno, so it might be confusing for beginners, especially the axetroy's is now deprecated.

Comment (piscisaureus) [reply docs/getting_started/setup_your_environment.md:46]: Maybe we have a bit of a lost in translation situation going on, but I think "the community" is more honest.
True, "we" *could* mean "the community", but I think it could be easily mistaken for us "the Deno maintainers" claiming we created those plugins. Which we definitely did not...

Comment (unbyte) [reply docs/getting_started/setup_your_environment.md:46]: 😂I think you are right

**Comment by lucacasonato** [comment]
Duplicate of #5292

**Comment by piscisaureus** [comment]
#5527
